project/Ydoc.scala

```diff
@@ -64,18 +69,16 @@ object Ydoc {
         Seq(resourceYdocServerJs)
     }
 
-    val ydocServerPolyglot = base / "app" / "ydoc-server-polyglot"
-    val ydocServerPolyglotSrc =
-      ydocServerPolyglot * ("*.mjs" | "*.json") +++ (ydocServerPolyglot / "src") ** "*.ts"
-    val ydocShared = base / "app" / "ydoc-shared"
-    val ydocSharedSrc =
-      ydocShared * ("*.ts" || "*.json") +++
-      (ydocShared / "src") ** "*.ts" +++
-      (ydocShared / "parser-codegen") ** "*.ts"
-    val sharedSrc  = (base / "app" / "gui2" / "shared") ** "*.ts"
-    val inputFiles = ydocServerPolyglotSrc +++ sharedSrc
-
-    generator(inputFiles.get)
+    val sourceFiles: PathFinder =
+      (base / "app") ** ("*.js" | "*.ts" | "*.json" | "*.rs" | "*.toml")
+    val nodeModulesFiles =
+      (base / "app") ** "node_modules" ** "*"
+    val ideDesktopFiles =
+      (base / "app") ** "ide-desktop" ** "*"
+
+    val inputFiles = sourceFiles --- nodeModulesFiles --- ideDesktopFiles
```

Comment:
Optimized build inputs a bit

Reply:
I cannot imagine what is the effect of this changes?

Reply:
There should be no difference. But this definition of build inputs is simpler